docs/timeseries/index.rst

```diff
@@ -0,0 +1,333 @@
+    2009-05-11T18:00:20.225  1.0238128e+06
+    2009-05-11T18:01:19.065  1.0243234e+06
+    2009-05-11T18:02:17.923  1.0244257e+06
+    2009-05-11T18:03:16.772  1.0248654e+06
+    2009-05-11T18:04:15.630  1.0250156e+06
+    2009-05-11T18:05:14.479  1.0254516e+06
+    2009-05-11T18:06:13.328  1.0254685e+06
+    2009-05-11T18:07:12.186  1.0259309e+06
+
+    >>> ts[0:4]  # doctest: +REMOTE_DATA
+    <TimeSeries length=4>
+              time             timecorr   ...   pos_corr1      pos_corr2
+                                  d       ...      pix            pix
+             object            float32    ...    float32        float32
+    ----------------------- ------------- ... -------------- --------------
+    2009-05-02T00:41:40.338  6.630610e-04 ...  1.5822421e-03 -1.4463664e-03
+    2009-05-02T00:42:39.188  6.630857e-04 ...  1.5743829e-03 -1.4540013e-03
+    2009-05-02T00:43:38.045  6.631103e-04 ...  1.5665225e-03 -1.4616371e-03
+    2009-05-02T00:44:36.894  6.631350e-04 ...  1.5586632e-03 -1.4692718e-03
+
+
+As seen in the example above, |TimeSeries| objects have a ``time``
+column, which is always the first column. This column can also be accessed using
+the ``.time`` attribute::
+
+    >>> ts.time  # doctest: +REMOTE_DATA
+    <Time object: scale='tdb' format='isot' value=['2009-05-02T00:41:40.338' '2009-05-02T00:42:39.188'
+      '2009-05-02T00:43:38.045' ... '2009-05-11T18:05:14.479'
+      '2009-05-11T18:06:13.328' '2009-05-11T18:07:12.186']>
+
+and is always a |Time| object (see :ref:`Times and Dates <astropy-time>`), which
+therefore supports the ability to convert to different time scales and formats::
+
+    >>> ts.time.mjd  # doctest: +REMOTE_DATA
+    array([54953.0289391 , 54953.02962023, 54953.03030145, ...,
+           54962.7536398 , 54962.75432093, 54962.75500215])
+
+    >>> ts.time.unix  # doctest: +REMOTE_DATA
+    array([1.24122483e+09, 1.24122489e+09, 1.24122495e+09, ...,
+           1.24206505e+09, 1.24206511e+09, 1.24206517e+09])
+
+Let's use what we've seen so far to make a plot
+
+.. plot::
+   :context: reset
+   :nofigs:
+
+    from astropy.utils.data import get_pkg_data_filename
+    filename = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+    from astropy.timeseries import TimeSeries
+    ts = TimeSeries.read(filename, format='kepler.fits')
+
+.. plot::
+   :include-source:
+   :context:
+
+   import matplotlib.pyplot as plt
+   plt.plot(ts.time.jd, ts['sap_flux'], 'k.', markersize=1)
+   plt.xlabel('Barycentric Julian Date')
+   plt.ylabel('SAP Flux (e-/s)')
+
+It looks like there are a few transits! Let's use
+:class:`~astropy.stats.BoxLeastSquares` to estimate the period, using a box with
+a duration of 0.2 days::
+
+    >>> import numpy as np
+    >>> from astropy import units as u
+    >>> from astropy.stats import BoxLeastSquares
+    >>> keep = ~np.isnan(ts['sap_flux'])  # doctest: +REMOTE_DATA
+    >>> periodogram = BoxLeastSquares(ts.time.jd[keep] * u.day,
```

Comment:
This is a good argument for wanting to get these periodogram wrappers in time for release... this would look *much* cleaner if you could just do `BoxLeastSquares(ts[keep])...`

Reply:
I agree! I'll work on this in a separate PR though.